windows.md

```diff
@@ -59,6 +59,7 @@ With those compatibility things out of the way, you're ready to start the system
    corepack enable
    corepack prepare pnpm@latest --activate
    pnpm setup
+   source "$USERPROFILE/.bash_profile"
```

Comment:
since [we'll switch from `$USERPROFILE` to `~`](https://github.com/upleveled/system-setup/pull/91#issuecomment-2264962880), we'll have a provlem if we keep this here - [it appears PowerShell doesn't support tilde](https://www.reddit.com/r/PowerShell/comments/14wdeih/tilde/)

we could move the Hyper install + configuration with Git Bash up earlier (eg. as a part of step 6 or a new step between step 6 and 7)

downside: the checking of the shell settings are further apart, and could become inconsistent.

Reply:
@ProchaLu decided to remove the `source` command from step 7 of the Windows guide in the latest commits